### ISSUE TITLE
Web component: Customizable 8-Bar IBM logo - Keyboard - When focus reaches to IBM logo, tool tip is not appearing.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#5062]
## Detailed description

Issue:
Keyboard navigation - When focus reaches to IBM logo, tool tip is not appearing.

Expected:
Tooltip should be accessible for keyboard user

**Comment by ariellalgilmore** [comment]
This looks like a duplicate for #4800
